### ISSUE TITLE
Logstash Web server returns `500 Internal Server Error` on aarch64 linux nodes

### ISSUE THREAD

**Issue by robbavey** [elastic/logstash#13444]
**Logstash information**:

Please include the following information:

1. Logstash version (e.g. `bin/logstash --version`)

Snapshot versions of Logtstash `7.16.0`, `8.0.0` and `8.1.0`

2. Logstash installation source (e.g. built from source, with a package manager: DEB/RPM, expanded from tar or zip archive, docker)
Tested on docker and from command line
3. How is Logstash being run (e.g. as a service/service manager: systemd, upstart, etc. Via command line, docker/kubernetes)

**JVM** (e.g. `java -version`):

Bundled JVM

**OS version** (`uname -a` if on a Unix-like system):

**Description of the problem including expected versus actual behavior**:

When attempting to access any of the Logstash API, an internal server error is returned, with no error message in the logs.
However, running in debug, the following error line is output:

```
[2021-11-23T16:46:55,224][DEBUG][logstash.agent           ] 2021-11-23 16:46:55 -0500 Read: #<NoMethodError: undefined method `>=' for nil:NilClass>
```

This appears to be related to [this](https://github.com/jruby/jruby/issues/6821) jruby error - it only manifests on `aarch64` nodes as far as I can tell.

Further testing has shown that downgrading to Puma to `4.x` resolves the error, but it persists in all versions of `5.x`. See https://logstash-ci.elastic.co/job/elastic+logstash+master+multijob-aarch64-docker-acceptance/flavor=full,nodes=aarch64/80/console for an example docker integration test using a downgraded version of puma

**Steps to reproduce**:

Requires `aarch64` enabled nodes, such as `t4g` graviton node type on AWS

 1. Run minimal pipeline
 2. Try and access `http://localhost:9600`
 3. Note 500: Internal Server error is returned.

**Provide logs (if relevant)**:

```
[2021-11-23T16:46:55,224][DEBUG][logstash.agent           ] 2021-11-23 16:46:55 -0500 Read: #<NoMethodError: undefined method `>=' for nil:NilClass>
```

**Comment by robbavey** [comment]
cc @roaksoax

**Comment by robbavey** [comment]
Fixed by #13447
